app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Kindly take a look at your association the belongs_to should be to the author also add the class name eg ``belongs_to :foreign_key, class_name: 'User'``

Reply:
I have renamed the author_id column to user_id, therefore, by default, it belongs to the user_id, and does not need any foreign key constraint, Kindly guide, please.

Reply:
@for-ashraf I totally agree with you on your case since you have used `user_id` it is not a must to add the constrains.